docs/configuration.md

```diff
@@ -114,6 +114,7 @@ information about each option that appears later in this page.
           age: 168h
           interval: 24h
           dryrun: false
+        readonly: false
```

Comment:
I just tried this, but it appears that it won't work with environment variables. `REGISTRY_STORAGE_MAINTENANCE_READONLY_ENABLED` doesn't work, because the Configuration struct defines "maintenance" as a map, and apparently the code that handles environment variable overrides can't handle an implicit nested map.

Given that limitation, what do you think about keeping it as-is?